assets/src/apps/delivery/layouts/deck/DeckLayoutView.tsx

```diff
@@ -42,13 +43,40 @@ const InjectedStyles: React.FC<{ css?: string }> = (props) => {
   return injected ? <style>{injected}</style> : null;
 };
 
+const getActualOwnerDetails = (expression: string, sequence: any) => {
```

Comment:
I think it would help (me at least) to name it `identifier: string` OR `identifierExpression: string` because expression by itself is a bit to broad